notebook/static/notebook/js/quickhelp.js

```diff
@@ -54,6 +59,8 @@ define([
             { shortcut: "Ctrl-Right", help:i18n.msg._("go one word right")  },
             { shortcut: "Ctrl-Backspace", help:i18n.msg._("delete word before")  },
             { shortcut: "Ctrl-Delete",    help:i18n.msg._("delete word after")  },
+            { shortcut: cmd_ctrl + "y",   help:i18n.msg._("redo")  },
```

Comment:
Is there a reason you're using `cmd_ctrl` here? This is in the non-Mac section where it will always be Ctrl.